### PR TITLE
Add Slurm sattach info

### PR DESCRIPTION
**Pull request by Carldkennedy**

### CODE REVIEW

File: hpc/scheduler/index.rst [hunk 1]
```diff
@@ -473,6 +473,27 @@ Slurm Command                        Description
                                      e.g. ``--cpus-per-task``. 
 ==================================== =======================================================================
 
+.. _sattach_interactive_bessemer:
+
+Rejoining an interactive job
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+If we lose connection to an interactive job, we can use the ``sattach`` command which attaches to a running Slurm job step.
+Just keep in mind that ``sattach`` doesn't work for external or batch steps, as their input and output channels aren't
```
Comment (jkwmoore) [inline hpc/scheduler/index.rst:481]:
```suggestion
Just keep in mind that ``sattach`` doesn't work for external or batch steps, as they aren't 
```

File: hpc/scheduler/index.rst [hunk 2]
```diff
@@ -783,6 +804,27 @@ Slurm Command                        Description
                                      e.g. ``--cpus-per-task``. 
 ==================================== =======================================================================
 
+.. _sattach_interactive_stanage:
+
+Rejoining an interactive job
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+If we lose connection to an interactive job, we can use the ``sattach`` command which attaches to a running Slurm job step.
+Just keep in mind that ``sattach`` doesn't work for external or batch steps, as their input and output channels aren't
```
Comment (jkwmoore) [inline hpc/scheduler/index.rst:812]:
```suggestion
Just keep in mind that ``sattach`` doesn't work for external or batch steps, as they aren't 
```

**Review by jkwmoore — CHANGES_REQUESTED** [review]
Besides my comments for a slight simplification of the warning about different job step types LGTM.

If happy with my suggestions, please merge with them.

**Comment by willfurnass** [comment]
Could we have just one section for both clusters?  The content is effectively the same.

**Comment by jkwmoore** [comment]
Agree that would be ideal, but as it stands that would need a large refactor of the bulk of the page (which is needed at some point.)

**Comment by willfurnass** [comment]
That's the kind of rebuttal I normally give you!  Fair point; let's refactor separately.

File: hpc/scheduler/index.rst [hunk 3]
```diff
@@ -473,6 +473,27 @@ Slurm Command                        Description
                                      e.g. ``--cpus-per-task``. 
 ==================================== =======================================================================
 
+.. _sattach_interactive_bessemer:
+
+Rejoining an interactive job
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+If we lose connection to an interactive job, we can use the ``sattach`` command which attaches to a running Slurm job step.
```
Comment (nikishe) [inline hpc/scheduler/index.rst:480]: Change "we" to "you" for consistency

File: hpc/scheduler/index.rst [hunk 4]
```diff
@@ -783,6 +804,27 @@ Slurm Command                        Description
                                      e.g. ``--cpus-per-task``. 
 ==================================== =======================================================================
 
+.. _sattach_interactive_stanage:
+
+Rejoining an interactive job
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+If we lose connection to an interactive job, we can use the ``sattach`` command which attaches to a running Slurm job step.
```
Comment (nikishe) [inline hpc/scheduler/index.rst:811]: Change "we" to "you" for consistency

**Pull request merged**